vivisect/impemu/platarch/arm.py

```diff
@@ -208,8 +208,7 @@ def runFunction(self, funcva, stopva=None, maxhit=None, maxloop=None, tmode=None
                             logger.debug(repr(e))
                             break
                         except Exception as e:
-                            if not self.getMeta('silent'):
-                                logger.warning("funcva: 0x%x opva: 0x%x:  %r   (%r) (in emumon prehook: %r)", funcva, starteip, op, e, self.emumon)
+                            logger.warning("funcva: 0x%x opva: 0x%x:  %r   (%r) (in emumon prehook: %r)", funcva, starteip, op, e, self.emumon)
```

Comment:
this is a mistake.  when attempting to emulate a string or known bad opcodes, you confuse the user with all sorts of "error"s which are just the result of isProbablyCode()

Reply:
See custom logging level above